src/main/haskell/kore/src/Kore/ASTUtils/AlphaCompare.hs

```diff
@@ -0,0 +1,108 @@
+{-|
+Module      : Kore.ASTUtils.AlphaCompare
+Description : Compare
+Copyright   : (c) Runtime Verification, 2018
+License     : NCSA
+Maintainer  : phillip.harris@runtimeverification.com
+Stability   : experimental
+Portability : portable
+-}
+
+module Kore.ASTUtils.AlphaCompare
+    ( alphaEq
+    ) where
+
+import Data.List
+
+import Kore.AST.Common
+import Kore.ASTUtils.SmartPatterns
+
+-- alphaEq compares terms modulo renaming of bound variables
```

Comment:
Does this work as documentation? Shouldn't it start with -- |?

Reply:
It's implementation details, which I don't think belong in Haddock docs.

Reply:
Fine, but this function is public, so it needs its own haddock documentation. Please add it.